api-reference/10 UI Components/dxHtmlEditor/1 Configuration/tableContextMenu.md

```diff
@@ -6,7 +6,19 @@ default: true
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Configures the table context menu settings.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+Users can right-click a table cell to invoke the table context menu. The context menu items allow users to perform operations listed below. Each operation is performed relative to the clicked cell's position:
+
+**Insert**
+
+- A header row
+- A row above / below
+- A column to the left / right
+
+**Delete**
+
+- A column
+- A row
+- The table
```

Comment:
A one-dimension list would be more clear:

- Insert a header row
- ...
- Delete a column
- Delete a row